cmd/protoc-gen-go-grpc/grpc.go

```diff
@@ -197,6 +197,9 @@ func genService(gen *protogen.Plugin, file *protogen.File, g *protogen.Generated
 	g.P("// ", clientName, " is the client API for ", service.GoName, " service.")
 	g.P("//")
 	g.P("// For semantics around ctx use and closing/ending streaming RPCs, please refer to https://pkg.go.dev/google.golang.org/grpc/?tab=doc#ClientConn.NewStream.")
+	g.P() // for vscode dosn't display top of comment
+	// print comment without '\n', if comment has '\n', vscode dosn't display it
+	g.P(strings.TrimSpace(service.Comments.Leading.String()))
```

Comment:
Suggestion: Add Comments about the call is intended to do Suggestion and may be Refactor to a separate Method for better readability

Reply:
ok, wait a moment